packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -162,7 +208,11 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
         this.props.xAxisType,
         this.props.barwidth!,
         this.props.tickValues!,
+        // This is only used for Horizontal Bar Chart with Axis for y as string axis
+        this.state.startFromX,
       ),
+      containerHeight: this.state.containerHeight - this.state._removalValueForTextTuncate!,
```

Comment:
is this line relevant to horizontal bar chart with axis

Reply:
No this is not related to HBCWA, this is the fix for wrap labels scenario for other charts. Let me know if i should do this in separate PR